src/lib/storage/chunk.hpp

```diff
@@ -41,7 +41,7 @@ class Chunk : private Noncopyable {
   std::shared_ptr<BaseSegment> get_segment(ColumnID column_id) const;
 
  protected:
-  // Implementation goes here
+  std::vector<std::shared_ptr<BaseSegment>> _segments = {};
```

Comment:
Not sure what the benefits of initialisation in this place are. Comparing with hyrise they also don't initialise in this place in hpp files. 

But coding style wise, if you decide to do the initialisation - make sure to make it consitent over all files e.g. comparing to storage_manager.hpp line 48.